src/react_native/core.cljs

```diff
@@ -20,7 +20,6 @@
 (defn image-get-size [uri callback] (.getSize ^js (.-Image ^js react-native) uri callback))
 (def text (reagent/adapt-react-class (.-Text ^js react-native)))
 (def text-input (reagent/adapt-react-class (.-TextInput ^js react-native)))
-
```

Comment:
could you pls rollback this file changes

Reply:
Done